.github/workflows/operate-update-docs-screenshots.yml

```diff
@@ -1,5 +1,38 @@
 name: Operate Update screenshots
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "stable/**"
+      - "release/**"
+    paths:
+      - ".ci/**"
+      - ".github/actions/**"
+      - ".github/workflows/operate-*"
+      - "bom/*"
+      - "distro/**"
+      - "operate.Dockerfile"
+      - "operate/**"
+      - "parent/*"
+      - "pom.xml"
```

Comment:
The Operate backend is mocked in this workflow ([example](https://github.com/camunda/camunda/blob/main/operate/client/e2e-playwright/docs-screenshots/process-instance-modification.spec.ts#L28-L40)), which makes it independent from changes in zeebe or Operate backend. I think it should only be triggered when there are frontend changes (or to the yaml file itself), same as for [frontend tests](https://github.com/camunda/camunda/blob/main/.github/workflows/operate-frontend.yml). 

Same for the pull_request trigger below.

Reply:
Makes sense, I didn't realize the BE was mocked and was thinking like the visual regression tests where a "external" change (BE, changes in carbon) would result in changes and it would be good for us to catch it. I've fixed it to match the FE tests. Good catch!